javascript.js

```diff
@@ -0,0 +1,195 @@
+const myArray = [
+  {
+    class: 'background-img1 background-img',
+    class2: 'card-img',
+    subtitle: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    popuptext: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.',
+    image: {
+      image1: './imgs/placeholder.png',
+      popup: './imgs/SnapshootPortfolio.png',
+    },
+    langlist: {
+      css: 'css',
+      html: 'html',
+      bootstrap: 'bootstrap',
+      ruby: 'Ruby',
+      lags: 'Ruby on rails',
+    },
+    btn: 'See Project',
+    style: {
+      style1: 'section-1',
+    },
```

Comment:
- [ ] Great job so far. please kindly remove the up/down scrolling from the mobile version
- [ ] please centralize the popup on the mobile version so it looks like that of figma
yours
![Screenshot (444)_LI](https://user-images.githubusercontent.com/65298048/189480580-6b6d7c0d-347a-4042-92f5-dd3dd7849fee.jpg)
figma
![Screenshot (443)](https://user-images.githubusercontent.com/65298048/189480588-88e5a002-da96-4c06-92fd-a0a11da4f810.png)